Extensions/ArtifactEngine/Providers/filesystemProvider.ts

```diff
@@ -89,7 +89,7 @@ export class FilesystemProvider implements models.IArtifactProvider {
             var items: models.ArtifactItem[] = [];
             fs.readdir(itemsPath, (error, files) => {
                 if (!!error) {
-                    Logger.logMessage("Unable to read directory " + itemsPath + ". Error: " + error);
+                    Logger.logError(tl.loc("UnableToReadDirectory", itemsPath, error));
```

Comment:
logging error will end up failing task. @omeshp is this fine?

Reply:
we should change this to logMessage.